module/VuFind/sql/mysql.sql

```diff
@@ -409,3 +409,25 @@ CREATE TABLE `access_token` (
   CONSTRAINT `access_token_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `login_token`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8mb4 */;
+CREATE TABLE `login_token` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` int NOT NULL,
+  `token` varchar(255) NOT NULL,
+  `series` varchar(255) NOT NULL,
+  `last_login` datetime NOT NULL,
+  `browser` varchar(255) NULL,
+  `platform` varchar(255) NULL,
+  `expires` int NOT NULL,
+  `last_session_id` varchar(255) NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `user_id_series` (`user_id`, `series`)
```

Comment:
I think there's a missing comma at the end of this line -- it's preventing the database from being created correctly in my test instance. I don't have write access to this branch, so I can't fix it myself. Do you mind taking a look when you have a moment?

Reply:
comma added

Reply:
Thanks! I'll take another look at this after @sturkel89 has had a chance to do some testing.